src/StackExchange.Redis/ServerSelectionStrategy.cs

```diff
@@ -114,19 +114,19 @@ public ServerEndPoint Select(Message message)
                     if (slot == MultipleSlots) throw ExceptionFactory.MultiSlot(multiplexer.IncludeDetailInExceptions, message);
                     break;
             }
-            return Select(slot, message.Command, message.Flags);
+            return Select(slot, message.Command, message.Flags, allowDisconnected);
         }
 
-        public ServerEndPoint Select(RedisCommand command, in RedisKey key, CommandFlags flags)
+        public ServerEndPoint Select(RedisCommand command, in RedisKey key, CommandFlags flags, bool allowDisconnected = false)
```

Comment:
Eliminate the `= false` defaults, so callers are always explicit about it?

Reply:
I was torn on this - wanted to make it obvious that _by default_ it should be `false` (I'd love an analyzer that requires putting the argument name everywhere it's used). No strong feelings here, but other approaches weren't as clean, e.g. flags on a message had overhead and don't work where we're passing just a command for things like server selection, so stuck with true/false down the pipe.